sdk/cmake/config.cmake

```diff
@@ -58,11 +58,11 @@ set(_ELF_ FALSE CACHE BOOL
 "Whether to compile support for ELF files.
 Do not enable unless you know what you're doing.")
 
-set(NSWPAT FALSE CACHE BOOL
-"Whether to build apps/libs with features covered by software patents.
-If you live in a country where software patents are valid/apply, don't
-enable this (except they/you purchased a license from the patent owner).
-This setting is disabled by default.")
+
```

Comment:
Why lines are blanked instead of just removing it? :thinking:

Reply:
to avoid change of line numbers

Reply:
No one cares about line numbers. But people care about meaningless empty lines.

Reply:
ok